docs/business.yaml

```diff
@@ -1564,15 +1564,604 @@ paths:
+                        inColinOnly: false
+                        isCorrected: false
+                        isCorrectionPending: false
+                        name: amalgamationApplication
+                        status: DRAFT
+                        submitter: mocked submitter
+                amalgamation-short-form-vertical-response:
+                  summary: Amalgamation Short Form Vertical Response
+                  value:
+                    filing:
+                      amalgamationApplication:
+                        amalgamatingBusinesses:
+                          - identifier: BC0883189
+                            role: holding
+                            type: lear
+                          - identifier: BC0883230
+                            role: amalgamating
+                            type: lear
+                        contactPoint: 
+                          email: "contact@example.com"
+                          phone: (555) 555-5555
+                        nameRequest: 
+                            legalType: BC
+                        parties:
+                          - mailingAddress:
+                              addressCity: Victoria
+                              addressCountry: Canada
+                              addressCountryDescription: Canada
+                              addressRegion: BC
+                              deliveryInstructions: ''
+                              postalCode: V8Z 5C5
+                              streetAddress: 3950 Helen Rd
+                              streetAddressAdditional: ''
+                            officer:
+                              email: null
+                              firstName: Completing
+                              lastName: Party
+                              partyType: person
+                            roles:
+                              - appointmentDate: '2024-07-24'
+                                roleType: Completing Party
+                        type: vertical
+                      business: 
+                        identifier: TO12345678
+                      header: 
+                        accountId: 3445
+                        affectedFilings: []
+                        availableOnPaperOnly: false
+                        certifiedBy: full name
+                        colinIds: []
+                        comments: []
+                        date: 2024-07-25T19:54:02.949741+00:00
+                        deletionLocked: false
+                        effectiveDate: 2024-07-25T19:54:02.949764+00:00
+                        email: no_one@never.get
+                        filingId: 150338
+                        inColinOnly: false
+                        isCorrected: false
+                        isCorrectionPending: false
+                        name: amalgamationApplication
+                        status: DRAFT
+                        submitter: mocked submitter
```

Comment:
Based on the new structure, added Registration example under `\businesses` path. Paul's ticket will add IA